screen/accounts/src/main/java/com/ivy/accounts/AccountsState.kt

```diff
@@ -12,5 +12,6 @@ data class AccountsState(
     val totalBalanceWithExcludedText: String,
     val totalBalanceWithoutExcluded: String,
     val totalBalanceWithoutExcludedText: String,
-    val reorderVisible: Boolean
+    val reorderVisible: Boolean,
+    val showCompactAccounts: Boolean
```

Comment:
rename to compactAccountsModeEnabled, "show" is unnecessary

Reply:
Done